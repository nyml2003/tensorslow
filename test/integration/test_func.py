def f(a, b):
    print(a, b) 
    a = a + b
    b = a - b
    a = a - b
    print(a, b)
    
f(114, 514)
