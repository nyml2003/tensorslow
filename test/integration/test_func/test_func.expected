114,514
514,114
